code/src/components/LoadingIndicator.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+
+export const LoadingIndicator = () => {
+  const isLoading = useSelector((state) => state.ui.isLoading)
+
+  return (
+    <>
+      {isLoading && <div>Loading ...</div>}
+    </>
+  )
+}
```

Comment:
Looks like you have everything set up for this loading indicator ⭐ Forgot to mount it? 🤔